Specifications/Events/for_EventStore/when_committing_a_public_object_with_an_event_type.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.SDK.Events.for_EventConverter.given;
+using Machine.Specifications;
+using It = Machine.Specifications.It;
+
+namespace Dolittle.SDK.Events.for_EventStore
+{
+    public class when_committing_a_public_object_with_an_event_type : given.an_event_store_and_an_execution_context
+    {
+        static an_event content;
+        static EventSourceId event_source;
+        static EventType event_type;
+
+        Establish context = () =>
+        {
+            content = new an_event("goodbye public world", 12345, true);
+            event_source = new EventSourceId(Guid.NewGuid());
+            event_type = new EventType(Guid.NewGuid());
+        };
```

Comment:
To be thorough, we could also check that if you send in an event of type that has been associated with an `EventType`, it doesn't use the registered one, but rather the one you provide as an argument.